### ISSUE TITLE
Toast description should wrap

### ISSUE THREAD

**Issue by ataravati** [GeekyAnts/NativeBase#4354]
** Before reporting this bug, please...**

1. Check the latest documentation: https://docs.nativebase.io/
2. Check for existing open/closed issues for a possible duplicate before creating a new issue: https://github.com/GeekyAnts/NativeBase/issues
3. Use the latest NativeBase release: https://github.com/GeekyAnts/NativeBase/releases
4. Check examples from NativeBase KitchenSink https://github.com/GeekyAnts/NativeBase-KitchenSink
5. For discussion purpose make use of NativeBase discord: https://discord.gg/TSgCw2UPmb

**Describe the bug**
If the description of the Toast doesn't fit in the container it should wrap. It overflows.

**To Reproduce**
Steps to reproduce the behaviour:

Just use a toast with a long description and you'll see it.

**Expected behaviour**

**CodeSandBox/Snack link**
It would be easier for us to debug if you can reproduce the error in a codesandbox link or a snack link. You can use the [codesandbox template](https://codesandbox.io/s/native-base-v3-template-mq6ox)/[snack template](https://snack.expo.io/@mdrehman/nativebase-template).

**Screenshots**

**Platform**

- NativeBase version [e.g. 3.0.1]
- Browser version [e.g. 22]
- React Native version [e.g. 0.64]
- Expo sdk version [e.g. 41]
- node [e.g. 14.16.1]
- yarn or npm [e.g. 6.1]

**Additional information**

1. Add any other context about the problem here.
2. Is the bug present in both iOS and Android or in any one of them?
3. If you want your issue to be looked at quicker, please attach a snack reproducible with your issue. Makes it easier for us!

**Comment by MD-REHMAN** [comment]
Unable to reproduce @ataravati the issue. Please provide simple snack for the same.

**Comment by ataravati** [comment]
@MD-REHMAN This is weird. I can't reproduce the issue in a snack.

Here's what I have in App.tsx:

```
      <Provider rootStore={rootStore}>
       <StripeProvider publishableKey={stripePublishableKey}>
          <SafeAreaProvider>
            <NativeBaseProvider theme={theme}>
              <NotificationsContainer>
                <BottomSheetModalProvider>
                  <Navigation colorScheme={colorScheme} />
                  <StatusBar />
                </BottomSheetModalProvider>
              </NotificationsContainer>
            </NativeBaseProvider>
          </SafeAreaProvider>
        </StripeProvider>
      </Provider>
```

The toast is displayed in the `NotificationsContainer` component that just returns `<>{children}</>`.


Here's the toast:

```   
   toast.show({
      title: "Test long message",
      description: "This is a very very long toast message that doesn't wrap correctly, because it's very very long.",
      status: "info",
      placement: "top",
    });
```

**Comment by Viraj-10** [comment]
Hi @ataravati ,
Issue Seems to be caused by one of the wrappers. I tried to replicate the scenario with different wrappers but still unable to reproduce the issue. If you can, please share a snack.

**Comment by surajahmed** [comment]
@ataravati This issue has been resolved in version `3.3.6`. If this issue still persists, please feel free to reopen it.
